lib/node_modules/@stdlib/stats/base/dists/binomial/kurtosis/src/main.c

```diff
@@ -0,0 +1,49 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/binomial/kurtosis.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/constants/float64/pinf.h"
+#include "stdlib/math/base/assert/is_nonnegative_integer.h"
+#include <stdint.h>
+
+/**
+* Returns the kurtosis of a binomial distribution.
+*
+* @param n  number of trials
+* @param p  success probability
+* @return   excess kurtosis
+*
+* @example
+* double y = stdlib_base_dists_binomial_kurtosis( 100, 0.1 );
+* // returns ~0.051
+*/
+double stdlib_base_dists_binomial_kurtosis( const int32_t n, const double p ) {
+	if (
+		stdlib_base_is_nan( n ) ||
+		stdlib_base_is_nan( p ) ||
+		n < 0.0 ||
+		n >= INT32_MAX ||
+		p < 0.0 ||
+		p > 1.0
+	) {
+		return 0.0 / 0.0;
+	}
+	double pq = p * ( 1.0 - p );
+	return ( 1.0 - (6.0 * pq ) ) / ( n * pq );
```

Comment:
```suggestion
	return ( 1.0 - ( 6.0 * pq ) ) / ( n * pq );
```

Use consistent spacing.